src/Config/Config.php

```diff
@@ -245,6 +245,21 @@ class Config
 
     const EMAIL_ALERTS_MODULE_NAME = 'ps_emailalerts';
 
+    const MOLLIE_VOUCHER_CATEGORY = 'MOLLIE_VOUCHER_CATEGORY';
+
+    const MOLLIE_VOUCHER_CATEGORY_NULL = 'null';
+    const MOLLIE_VOUCHER_CATEGORY_MEAL = 'meal';
+    const MOLLIE_VOUCHER_CATEGORY_GIFT = 'gift';
+    const MOLLIE_VOUCHER_CATEGORY_ECO = 'eco';
+
+    const MOLLIE_VOUCHER_ATTRIBUTE_ID = 'MOLLIE_VOUCHER_ATTRIBUTE_ID';
+    const MOLLIE_VOUCHER_ATTRIBUTE = 'MOLLIE_VOUCHER_ATTRIBUTE_';
+    const MOLLIE_VOUCHER_CATEGORIES = [
```

Comment:
"Food and drinks" sounds like something shown to the user. Make sure you don't forget to translate it if it's relevant.

Reply:
Its used to create feature, you can translate it later in PS feature tab.